db/sqldb_test.go

```diff
@@ -382,16 +381,46 @@ func TestGetMTASTSStats(t *testing.T) {
 			t.Fatal(err)
 		}
 	}
-	result, err := database.GetMTASTSStats("domains-depot")
+	result, err := database.GetStats("domains-depot")
```

Comment:
Now that we have more than one Source type, let's put an enum representing the two possible sources (I think `TOP_DOMAINS` and `local` right now?

Reply:
I like this a lot better, thanks.